src/Aqua.jl

```diff
@@ -63,6 +73,7 @@ function test_all(
     deps_compat = true,
     piracies = true,
     persistent_tasks = true,
+    undocumented_names = true,
```

Comment:
Setting this to true implies that dependents of this package might now fail their tests, right? In that case, shouldn't this be a breaking change, i.e. up to `v0.9.1` instead of `v0.8.10`?

Reply:
I don't know what Aqua's versioning policy is when adding a new test. I'll defer to the maintainers